src/routes/artwork/[id].svelte

```diff
@@ -63,7 +63,7 @@
 
   $: setup(id);
   let setup = async () => {
-    query(getArtwork(id))
+    query(getArtwork(id, $user.id))
```

Comment:
We need to handle the case when $user.id is not defined because someone is browsing the artwork without being logged in or necessarily having an account